src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,72 @@
 'use strict';
 
+const NAME = '';
+const HEALTH = 100;
+
 class Animal {
-  // write your code here
+  static alive = [];
+
+  health;
+  name;
+
+  constructor(name = NAME, health = HEALTH) {
+    this.name = name;
+    this.health = health;
+
+    if (this.health > 0) {
+      Animal.alive.push(this);
```

Comment:
Consider checking for `this.health > 0` inside the `takeDamage` method instead of the constructor. It's more robust to have the check where the health is actually modified.